goal_src/jak2/engine/gfx/mood/mood-tables2.gc

```diff
@@ -7,3 +7,116 @@
+                                         :fog-dists (new 'static 'vector :x 262144.0 :y 3690496.0 :z 255.0 :w 64.0)
+                                         :erase-color (new 'static 'vector :w 128.0)
+                                         )
+                                       (new 'static 'mood-fog
+                                         :fog-color (new 'static 'vector :x 160.0 :y 150.0 :z 200.0 :w 128.0)
+                                         :fog-dists (new 'static 'vector :x 262144.0 :y 3690496.0 :z 255.0 :w 64.0)
+                                         :erase-color (new 'static 'vector :w 128.0)
+                                         )
+                                       (new 'static 'mood-fog
+                                         :fog-color (new 'static 'vector :x 16.0 :y 32.0 :z 100.0 :w 128.0)
+                                         :fog-dists (new 'static 'vector :x 262144.0 :y 3690496.0 :z 255.0 :w 64.0)
+                                         :erase-color (new 'static 'vector :w 128.0)
+                                         )
+                                       (new 'static 'mood-fog
+                                         :fog-color (new 'static 'vector :y 24.0 :z 64.0 :w 128.0)
+                                         :fog-dists (new 'static 'vector :x 262144.0 :y 3690496.0 :z 255.0 :w 64.0)
+                                         :erase-color (new 'static 'vector :w 128.0)
+                                         )
+                                       (new 'static 'mood-fog
+                                         :fog-color (new 'static 'vector :y 56.0 :z 72.0 :w 128.0)
+                                         :fog-dists (new 'static 'vector :x 262144.0 :y 3690496.0 :z 255.0 :w 64.0)
+                                         :erase-color (new 'static 'vector :w 128.0)
+                                         )
+                                       )
+                               )
+  )
+
+(defun init-overide-table ((arg0 mood-table))
+  (set! (-> arg0 mood-fog-table) (new 'debug 'mood-fog-table))
+  (set! (-> arg0 mood-color-table) (new 'debug 'mood-color-table))
+  (set! (-> arg0 mood-channel-group) *no-cloud-mood-channel-group*)
+  (set! (-> arg0 mood-direction-table) *mood-direction-table*)
+  (set! (-> arg0 mood-sky-table) *mood-sky-table*)
+  (set! (-> arg0 mood-interp-table) (the-as basic *mood-interp-table*))
+  (mem-copy! (the-as pointer (-> arg0 mood-fog-table)) (the-as pointer *no-cloud-clear-mood-fog-table*) 384)
+  (mem-copy! (the-as pointer (-> arg0 mood-color-table)) (the-as pointer *no-cloud-mood-color-table*) 256)
+  (none)
+  )
+
+(define *overide-table* (new 'static 'mood-table))
+
+(init-overide-table *overide-table*)
+
+;; ERROR: function has no type analysis. Cannot decompile.
+
+;; ERROR: function has no type analysis. Cannot decompile.
+
+;; ERROR: function has no type analysis. Cannot decompile.
```

Comment:
are these the broken ones due to far labels/ash stuff?

Reply:
nah just missed them